selfdrive/car/honda/values.py

```diff
@@ -1301,4 +1303,4 @@ class CAR:
 }
 
 HONDA_BOSCH = set([CAR.ACCORD, CAR.ACCORDH, CAR.CIVIC_BOSCH, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_5G, CAR.CRV_HYBRID, CAR.INSIGHT, CAR.ACURA_RDX_3G])
-HONDA_BOSCH_ALT_BRAKE_SIGNAL = set([CAR.ACCORD, CAR.CRV_5G, CAR.ACURA_RDX_3G])
+HONDA_BOSCH_ALT_BRAKE_SIGNAL = set([CAR.ACCORD, CAR.CRV_5G, CAR.ACURA_RDX_3G])
```

Comment:
suggest restoring the file's terminating newline-- there shouldn't be a change here

Reply:
yeah, let's revert this

Reply:
Weird, not sure how that happened. I'm still relatively new to this, but all I did was input 2 new lines in Textmate and save it. 

Should I retry and reupload to see if it works correctly?